libraries/AP_NavEKF3/AP_NavEKF3.h

```diff
@@ -445,7 +445,7 @@ class NavEKF3 {
     const float gndEffectBaroScaler = 4.0f;        // scaler applied to the barometer observation variance when ground effect mode is active
     const uint8_t gndGradientSigma = 50;           // RMS terrain gradient percentage assumed by the terrain height estimation
     const uint16_t fusionTimeStep_ms = 10;         // The minimum time interval between covariance predictions and measurement fusions in msec
-    const uint8_t sensorIntervalMin_ms = 50;       // The minimum allowed time between measurements from any non-IMU sensor (msec)
+    const uint8_t sensorIntervalMin_ms = 20;       // The minimum allowed time between measurements from any non-IMU sensor (msec)
```

Comment:
This one will need evaluation in order to check that we don't speed to much time to  handle wheel encoder in depend of the rest

Reply:
I had tried the original value of 50. There is a visible performance boost when reduced to 20. But I am not sure if there are some unseen consequences for other sensors. Nothing I could see though.